### PR TITLE
test(AnimateBlock): test for AnimateBlock added

### PR DESCRIPTION
**Pull request by niktverd**

### CODE REVIEW

File: src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    test('add className', () => {
+        const className = 'my-class';
+
+        render(<AnimateBlock qa={qaId} className={className} />);
+        const component = screen.getByTestId(qaId);
+
+        expect(component).toHaveClass(className);
+    });
+
+    test('add className', () => {
```
Comment (yuberdysheva) [inline src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx:27]: this is repeated twice

File: src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+describe('AnimateBlock', () => {
+    test('render AnimateBlock by default', async () => {
+        render(<AnimateBlock qa={qaId} />);
+
+        const component = screen.getByTestId(qaId);
+        expect(component).toBeInTheDocument();
+    });
+
+    test('add className', () => {
+        const className = 'my-class';
```
Comment (yuberdysheva) [inline src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx:19]: Can we use here a variable? We can use it in other tests

File: src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+        render(<AnimateBlock qa={qaId} className={className} />);
+        const component = screen.getByTestId(qaId);
+
+        expect(component).toHaveClass(className);
+    });
+
+    test('use passed style', () => {
+        const style = {color: 'red'};
```
Comment (yuberdysheva) [inline src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx:37]: Maybe (I'm mot sure) it is also can be a common variable

File: src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+        render(<AnimateBlock style={style} qa={qaId} />);
+        const component = screen.getByTestId(qaId);
+
+        expect(component).toHaveStyle(style);
+    });
+
+    test('call onUpdate with checked status', async () => {
```
Comment (yuberdysheva) [inline src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx:45]: What does it means this test?

**Review by yuberdysheva — CHANGES_REQUESTED** [review]

File: src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx (hunk 1 above)
Comment (niktverd) [reply src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx]: I removed it. Thank you

File: src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx (hunk 2 above)
Comment (niktverd) [reply src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx:19]: Let us leave it as it is

File: src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx (hunk 3 above)
Comment (niktverd) [reply src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx:28]: Let us leave it as it is

File: src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx (hunk 4 above)
Comment (niktverd) [reply src/components/AnimateBlock/__tests__/AnimateBlock.test.tsx]: It detects that onScroll event is fired. Actually, after your comment I thought twice and rewrote this test. Please check it one more time

**Review by yuberdysheva — APPROVED** [review]

**Pull request merged**